### DISCUSSION TITLE
failed to publish to npm with the reason "Must associate a single dist tag with new publishes"

### DISCUSSION THREAD

**Discussion by plainheart** [community/community#128429] [npm]
---UPDATE---
I've solved this issue by adjusting the time of publishing the two versions.
\---------------

Hi, dear npm team, 

We have a GitHub action that automatically publishes nightly versions for the `zrender` library. Most of the time it works well but sometimes it will fail for the following reason:

```
npm ERR! code E400
npm ERR! 400 Bad Request - PUT https://registry.npmjs.org/zrender-nightly - Must associate a single dist tag with new publishes.
```

I searched in the documentation and Google but found nothing useful. I guess it might be due to the near time? The `next` version and `latest` version will almost be published to npm at the same time. Perhaps, I should wait for a moment to publish another tag after a tag is published?

The failed runs can be found here: https://github.com/ecomfe/zrender/actions?query=is%3Afailure

Regards.

**Comment by marcbachmann** [comment]
We're observing the same issue when we backport fixes and they get released within the same time.
With the github automerge this issue got even more prominent.
